src/runtime/Loader.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+export type LoaderProps = { errors: React.ReactNode[] };
+interface Props {
+  render: React.ComponentProps<any>;
```

Comment:
`any` isn't right here, we want RenderProps. Please try and never use `any` as much as possible.

Reply:
Also isn't React.ComponentProps not what we want? Don't we want `(props: RenderProps) => ReactElement`?